thread/stack-allocator.cpp

```diff
@@ -169,8 +173,13 @@ size_t pooled_stack_trim_threshold(size_t x) {
     return get_pooled_stack_allocator().threshold(x);
 }
 
-size_t pooled_stack_trim_current_vcpu(size_t keep_size);
-size_t pooled_stack_trim_threshold(size_t x);
+void use_pooled_stack_allocator() {
+    get_pooled_stack_allocator();
+    std::call_once(init_once_flag, [] {
```

Comment:
Since it is a one-way setter and always put same value, is it really need to use a `std::call_once`?